test/assignmentsix/Testuser.java

```diff
@@ -35,19 +39,37 @@ public void run(String outputPath) {
 		// Initialize Coordinator with InternalComputeEngine
 		UScomputerEngineConstructer coordinator = new UScomputerEngineConstructer(computeEngine, dataStore);
 
-		Testuser testuser = new Testuser(coordinator);
-
+//		Testuser testuser = new Testuser(coordinator); <- not sure what this line is doing 
 //		TODO: Call the Coordinator's methods for parsing the file and computing the data
 
+		// added methods to get List<Integer> from file and send it to internal compute
 		File inputFile = new File(inputPath);
-
 		coordinator.setInputFile(inputFile);
+		coordinator.setData();
+		List<Integer> computedResults = coordinator.runInternalCompute(coordinator.getData());
 
-//		List<Integer> parsedIntegers = coordinator.parseInputFile();
+		writeResultsToFile(computedResults, outputPath);
 
-//		Foo results = Call the compute engine to do the calculation
+		
 
+
+		
+//		List<Integer> parsedIntegers = coordinator.parseInputFile();
+//		Foo results = Call the compute engine to do the calculation
 //		coordinator.writeToFile(outputFile, results)
 	}
 
+	private void writeResultsToFile(List<Integer> results, String outputPath) {
```

Comment:
There's already functionality for this in DataStore, you should just make a request to do that instead.